extensions/amp-sticky-ad/amp-sticky-ad.md

```diff
@@ -38,11 +38,12 @@ limitations under the License.
 ## Behavior
 
 - There can be only one `<amp-sticky-ad>` in an AMP document. The `<amp-sticky-ad>` should only have one direct child of `<amp-ad>`.
-- The sticky will appear on the bottom of a page.
+- The sticky ad will appear on the bottom of a page.
 - The height of the sticky-ad is whatever its child needs up to its max-height.
 - The max-height of the sticky-ad is 100px, if the height exceeds 100px then the height would be 100px and overflow content will be hidden.
 - The width of the sticky-ad is set to 100% using CSS and cannot be overridden.
 - The sticky-ad will display after scroll one viewport height from top provided there is at least one more viewport of content available.
```

Comment:
Suggest adding new bullet : 
"When scrolled to the bottom of the page, the viewport is automatically padded with the additional height of the sticky ad, so that no content is ever hidden."